internal/wrphandlers/mocktr181/handler.go

```diff
@@ -173,32 +178,37 @@ func (h Handler) get(names []string) (int64, []byte, error) {
 					Value:      mockParameter.Value,
 					DataType:   mockParameter.DataType,
 					Attributes: mockParameter.Attributes,
+					Message:    "Success",
+					Count:      1,
 				})
 			default:
 				result.Parameters = append(result.Parameters, Parameter{
 					Message: fmt.Sprintf("Invalid parameter name: %s", mockParameter.Name),
 				})
-				statusCode = 520
+				result.StatusCode = 520
 			}
 		}
 	}
 
+	if len(result.Parameters) == 0 {
```

Comment:
If you default the `StatusCode` to `520` and only set it when appending to the parameters on line 171, then no conditional check is needed here.